src/plugins/plugin-http.js

```diff
@@ -76,8 +76,22 @@ function patchRequest (http, api) {
       var req = request.call(this, options, function(res) {
         api.wrapEmitter(res);
         var numBytes = 0;
-        res.on('data', function (chunk) {
-          numBytes += chunk.length;
+        var listenerAttached = false;
+        // Responses returned by http#request are yielded in paused mode. By
+        // attaching a 'data' listener to the request the http instrumentation was
+        // switching the stream to flowing mode which could cause the request to
```

Comment:
nit: Phrase the comment in present tense. A future reader of the comment may not have to context about what used to happen. The comment should instead explain what is the intent of code as present.